roles/ups/templates/unifiedpushserver.yml.j2

```diff
@@ -0,0 +1,6 @@
+apiVersion: push.aerogear.org/v1alpha1
```

Comment:
@grdryn do you know what's the reason for passing the `ups_server_name` here? Without that we could import this file rather than templating.

Reply:
I don't know much about this mobile-services-installer, so I'm not sure how important being able to customize the CR name is for that. All resources need names though, so it either needs to be templated or hardcoded.

One thing that it will affect a bit is the route that UPS will get.